mlir/include/air/Util/Dependency.h

```diff
@@ -56,9 +56,6 @@ Value getLoopCarriedTokenFromScfOp(scf::ParallelOp op);
 Value getLoopCarriedTokenFromScfOp(scf::ForOp op,
                                    std::string operand_or_argument = "operand");
 void addAsyncDependencyIfNew(air::AsyncOpInterface op, Value token);
-std::string getMemorySpaceAsString(Value memref);
-air::ChannelGetOp getTheOtherChannelOpThroughSymbol(air::ChannelPutOp put);
```

Comment:
Could leave the declaration of getTheOtherChannelOpThroughSymbol() here, as runner includes Dependency.h and makes use of those helper functions.

Reply:
Ok I see that the declaration is under Utils.h instead. This is okay and the PR looks good to me. Thanks